jhannett_rios/task/mission_to_mars/src/Simulation.java

```diff
@@ -0,0 +1,99 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Simulation {
+    private ArrayList<Item> items;
+    public Simulation(){
+    }
+    public void loadItems(Scanner phase) throws Exception{
+        items = new ArrayList<>();
+        while ((phase.hasNextLine())){
+            String currentLine = phase.nextLine();
+            String[] currentItem = currentLine.split("=");
+            Item item = new Item(currentItem[0], Double.parseDouble(currentItem[1]));
+            items.add(item);
+        }
+        phase.close();
+    }
+
+    public void loadU1(){
+        ArrayList<Rocket> rockets = new ArrayList<>();
+        int numOfRockets = 0;
+        while (this.items.size() > 0){
+            RocketU1 u1 = new RocketU1();
+            System.out.println("=         Load Items       =");
+            System.out.println("============================");
+            for(int i = this.items.size() - 1; i >= 0; i--){
+                if(u1.canCarry(this.items.get(i))){
+                    u1.addCarry(this.items.get(i));
+                    System.out.println("  " + items.get(i).getName() + " " + items.get(i).getWeight());
```

Comment:
This line is very long maybe you could break it like:
System.out.println("  " + items.get(i).getName() + " " 
                                 + items.get(i).getWeight());
In this link(https://www.oracle.com/technetwork/java/codeconventions-150003.pdf), you could review the Java convention.